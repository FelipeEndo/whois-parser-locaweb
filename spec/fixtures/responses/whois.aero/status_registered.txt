Domain Name: SRILANKAN.AERO
Registry Domain ID: D108300000000000246-AGRS
Registrar WHOIS Server:
Registrar URL:
Updated Date: 2021-01-08T12:04:52Z
Creation Date: 2002-03-26T06:48:27Z
Registry Expiry Date: 2022-03-26T06:48:27Z
Registrar Registration Expiration Date:
Registrar: Tuonome.it.srl d/b/a APIsrs.com
Registrar IANA ID: 380
Registrar Abuse Contact Email:
Registrar Abuse Contact Phone:
Reseller:
Domain Status: clientTransferProhibited https://icann.org/epp#clientTransferProhibited
Registrant Organization: SriLankan Airlines Ltd
Registrant State/Province: WP
Registrant Country: LK
Name Server: NS-1046.AWSDNS-02.ORG
Name Server: NS-2011.AWSDNS-59.CO.UK
Name Server: NS-133.AWSDNS-16.COM
Name Server: NS-877.AWSDNS-45.NET
DNSSEC: unsigned
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2021-11-09T15:49:32Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Access to WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the registry database. The data in this record is provided by The Registry Operator for informational purposes only, and accuracy is not guaranteed.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to (a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Registry Operator reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy.
The Registrar of Record identified in this output may have an RDDS service that can be queried for additional information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
