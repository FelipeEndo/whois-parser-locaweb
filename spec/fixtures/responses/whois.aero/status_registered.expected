#disclaimer
  %s == "Access to WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the registry database. The data in this record is provided by The Registry Operator for informational purposes only, and accuracy is not guaranteed.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to (a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Registry Operator reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy. The Registrar of Record identified in this output may have an RDDS service that can be queried for additional information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."

#domain
  %s == "srilankan.aero"

#domain_id
  %s == "D108300000000000246-AGRS"


#status
  %s == ["clientTransferProhibited https://icann.org/epp#clientTransferProhibited"]

#available?
  %s == false

#registered?
  %s == true

#reserved?
  %s == false


#created_on
  %s %CLASS{time}
  %s %TIME{2002-03-26 06:48:27 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2021-01-08 12:04:52 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2022-03-26 06:48:27 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "380"
  %s.name         == "Tuonome.it.srl d/b/a APIsrs.com"
  %s.url          == ""

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == nil
  %s[0].name          == nil
  %s[0].organization  == "SriLankan Airlines Ltd"
  %s[0].address       == nil
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == "WP"
  %s[0].country_code  == "LK"
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == nil

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{0}

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{0}

#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns-1046.awsdns-02.org"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns-2011.awsdns-59.co.uk"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns-133.awsdns-16.com"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns-877.awsdns-45.net"
